libipl/libipl.C

```diff
@@ -359,3 +362,43 @@ void ipl_error_callback(const ipl_error_info& error)
 		return;
 	g_ipl_error_callback_fn(error);
 }
+
+void ipl_process_fapi_error(const fapi2::ReturnCode& fapirc,
```

Comment:
Move these helper function under p10 folder. also you can define header file to support this or keep it in common.C

Reply:
JS: what about ipl_plat_error.C|H ? we might add more in the future right?